PolSourceTasks/src/classes/EditionTriggerTest.cls

```diff
@@ -50,13 +49,13 @@ private class EditionTriggerTest {
         
         //when
         for(Integer i = 0; i<3; i++){
```

Comment:
Wywal fora jak zmieniłeś na 0, nie ma sensu, żeby się to 3 razy przestawiało na true ;)